MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include full_text_function.sql
+include *requirements*.txt
+include CHANGELOG
```

Comment:
Including the changelog but not the readme?

Reply:
added now